src/main/java/pl/touk/sputnik/engine/score/ScoreStrategy.java

```diff
@@ -0,0 +1,82 @@
+package pl.touk.sputnik.engine.score;
+
+import com.google.common.collect.ImmutableSet;
+import org.jetbrains.annotations.NotNull;
+import pl.touk.sputnik.review.Review;
+import pl.touk.sputnik.review.Severity;
+
+import java.util.Set;
+
+public enum ScoreStrategy {
```

Comment:
The intention was to use implement a strategy pattern with enums, since there's no state so there only needs to be one instance of each one. I can revert this to ScoreStrategy being an interface with 4 implementations if you prefer. You no longer need a builder because there's nothing to build.

The advantage of this is that you can still do logging. Yeah that's a good advantage. I will do that.

Reply:
Thanks!